site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -365,13 +365,21 @@ include::../shared/_advanced-edition-blurb-api.adoc[]
 +
 :advanced_feature!:
 
+[field]#tenant.failedAuthenticationConfiguration.actionCancelPolicy.onPasswordReset# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.42.0#::
+Indicates if you want the user to be able to self-service unlock their account prior to the action duration by completing a password reset workflow.
```

Comment:
```suggestion
Indicates whether you want the user to be able to self-service unlock their account prior to the action duration by completing a password reset workflow.
```

Reply:
Missed this before I merged. Will make this change thanks!